frontend/src/metabase-lib/lib/queries/structured/MBQLClause.js

```diff
@@ -115,6 +115,14 @@ export class MBQLObjectClause {
   metadata() {
     return this._query.metadata();
   }
+
+  raw() {
```

Comment:
Nit: from the name of this function, at least for me it's not easy to understand what it accomplishes.

Does it make the entries raw?

Does it filter by only the raw entries?

Reply:
There is a method doing the same thing above in `MBQLArrayClause` class and is called `raw` too, so I tried to keep things consistent. I agree the naming could be better, but I'd like to avoid as many refactorings as possible for bug fixes before 41 is rolled out